### PR TITLE
Reset the Prepublishing nudges view when navigating between views

### LINKED ISSUE #15531: Prepublishing bottom sheet gets cut at the bottom after adding a category
### Expected behavior
I expect the prepublishing bottom sheet to show all options on screen 

### Actual behavior
The bottom sheet is cut at the bottom 
Simulator Screen Shot - iPhone 11 Pro - 2020-12-16 at 13 53 14 (1)

### Steps to reproduce the behavior
1. Start a new post
2. Tap publish 
3. See the prepublishing bottom sheet 
4. tap Categories
5. Add a category 
6. Go back 
7. See the bottom sheet cut at the bottom

### PR DESCRIPTION
**Pull request by emilylaguna**
Fixes #15531 

### To Test:

Follow the steps on Related PR: https://github.com/wordpress-mobile/WordPressUI-iOS/pull/81

### PR submission checklist:

- [x] I have considered adding unit tests where possible.
- [x] I have considered adding accessibility improvements for my changes.
- [x] I have considered if this change warrants user-facing release notes and have added them to `RELEASE-NOTES.txt` if necessary.

### CODE REVIEW

File: WordPress/Classes/ViewRelated/Post/Prepublishing Nudge/PrepublishingNavigationController.swift [hunk 1]
```diff
@@ -24,14 +24,32 @@ class PrepublishingNavigationController: LightNavigationController {
...
             }
 
             return visibleViewController.preferredContentSize
         }
     }
 
+    override func pushViewController(_ viewController: UIViewController, animated: Bool) {
+        super.pushViewController(viewController, animated: animated)
+
+        if let bottomSheet = self.parent as? BottomSheetViewController, let presentedVC = bottomSheet.presentedVC {
```
Comment (yaelirub) [inline WordPress/Classes/ViewRelated/Post/Prepublishing Nudge/PrepublishingNavigationController.swift]: How about moving this to a private function:
```
private func transition() {
        if let bottomSheet = self.parent as? BottomSheetViewController, let presentedVC = bottomSheet.presentedVC {
            presentedVC.transition(to: .collapsed)
        }
    }
```
And avoiding the duplication?

**Review by yaelirub — APPROVED** [review]
GtG after the comment is addressed

File: WordPress/Classes/ViewRelated/Post/Prepublishing Nudge/PrepublishingNavigationController.swift (hunk 1 above)
Comment (emilylaguna) [reply WordPress/Classes/ViewRelated/Post/Prepublishing Nudge/PrepublishingNavigationController.swift]: Good plan! 👍

Comment (emilylaguna) [reply WordPress/Classes/ViewRelated/Post/Prepublishing Nudge/PrepublishingNavigationController.swift]: Done in: https://github.com/wordpress-mobile/WordPress-iOS/pull/15551/commits/deeabfd699c0e53c9c19989f2cf481fc666a5aa4

**Pull request merged**